### DISCUSSION TITLE
Is there an indeterminate progress bar in WIX tool set?

### DISCUSSION THREAD

**Discussion by saadsaeed01** [wixtoolset/issues#8073] [Questions]
Hello 

My requirement is that I need an indeterminate progress bar

In ProgressBar control, I don't see any option to make it indeterminate

`<Control Id="ProgressBar" Type="ProgressBar"  X="20" Y="115"  Width="330" Height="10"  Text="!(loc.ProgressDlgProgressBar)">
				<Subscribe Event="SetProgress" Attribute="Progress"  />
</Control>`

is there a solution to this problem?

**Comment by robmen** [comment]
Pretty sure you'll need to open a feature request with the Windows Installer team to [add that option to their current supported list](https://learn.microsoft.com/en-us/windows/win32/msi/progressbar-control).

**Comment by saadsaeed01** [comment]
Thanks @robmen 

I don't have time to wait for this but thank you for your response!

**Reply by chrpai** [reply]
All of us would be 6' under before the Windows Installer team implemented it.
